ArduPlane/Parameters.cpp

```diff
@@ -1293,6 +1293,13 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPINFO(guidedHeading, "GUIDED_", 28, ParametersG2, AC_PID),
 #endif // OFFBOARD_GUIDED == ENABLED
 
+    // @Param: STALL_DETECT
+    // @DisplayName: Stall detection type
+    // @Description: Stall detection type. Different aircraft stall in different ways.
+    // @User: Advanced
+    // @Bitmask: 0:TECSbadDescent,1:SinkRate*2Max,2:SinkRate*4Max,3:RollError20deg,4:RollError30deg,5:RollError45deg,6:PitchError10deg,7:PitchError20deg,8:PitchError20deg,9:PitchError30deg,10:PitchError40deg,11:AltError10m,12:AltError20m,13:AltError40m,14:AltError60m
```

Comment:
I like this method because those flags light up in the log and it allows you to find out what thresholds to set for your own aircraft. Also, we should probably move away from angle error and move to rate error